src/Social.js

```diff
@@ -0,0 +1,83 @@
+import React, {Component, Children, createElement} from 'react'
+import classNames from 'classnames'
+import Button from "./Button"
```

Comment:
You'll be able to kill off this import if you follow one of the other comments I made.